django/contrib/postgres/fields/citext.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models import CharField
+
+__all__ = ['CitextField']
+
+
+class CitextField(CharField):
```

Comment:
(It's kinda hilarious to recommend a case change on a name that's literally case insensitive.)